docs/kb/semgrep-cloud-platform/saml-attributestatement.md

```diff
@@ -0,0 +1,30 @@
+---
+description: Solve an AttributeStatement issue encountered when setting up Semgrep with SSO.
+tags:
+  - Semgrep Cloud Platform 
+  - SSO
+  - Troubleshooting
+---
+
+# SAML configuration error: There is no AttributeStatement on the Response
+
+When setting up your SAML SSO (single sign-on) in Semgrep Cloud Platform, you may encounter the following configuration error: `There is no AttributeStatement on the Response`. This error can be seen after trying to configure SSO in Semgrep.
+
+![SAML AttributeStatement error](/img/kb/attribute-statement.png)
+
+This issue is caused by an **attribute** within your service provider (SP) that does not have a value ascribed to it. When configuring SSO, the payload you send back to your SP, such as Okta or OneLogin, contains this attribute and hence, the SP raises this error.
+
+## If you don't know the SAML attribute that is causing the error
+
+If you don't know what attribute the payload is failing on, you can investigate the payload to determine the attribute. Refer to [How to view a SAML response in your browser](https://support.okta.com/help/s/article/How-to-View-a-SAML-Response-in-Your-Browser-for-Troubleshooting?language=en_US) for debugging steps.
+
+After finding out what attribute is causing the error, proceed to the following section.
+
+## If you know the SAML attribute that is causing the error
+
+There are two possible solutions for this problem, depending on the expectations of your service provider (SP):
+
+- If your service provider anticipates a value for the specific SAML Attribute statement, you must include a value within the SAML settings. [Where do we include this value? From the service provider?]
```

Comment:
@fuzzmaestro1 For this line, can you let me know where we're expected to put the Attribute Statement? (I assume it's in the SP)

Reply:
@s-santillan The SAML provider, the IdP (not the SP) is responsible for the attributes.  
The service provider (SP)is the Semgrep App, but it is not responsible for the attributes.  The attributes are populated in the Identity Provider (IdP, such as Okta, OneLogin etc).  If the attribute in the IdP is not configured or misconfigured, this is manifested as a kind of AttributeStatement error is thrown by the SP (Semgrep in this case).